### ISSUE TITLE
Cannot resolve hidden-args after using Morganey 0.0.1 from JCenter

### ISSUE THREAD

**Issue by rexim** [morganey-lang/Morganey#271]
Blocks #261
### Reproduction steps
- Write a small sbt project that uses Morgnaey 0.0.1 as a dependency:

``` scala
resolvers += Resolver.jcenterRepo

libraryDependencies ++= Seq(
  "me.rexim" %% "morganey" % "0.0.1"
)

```
- Build the project
### Observed

```
[error] (*:update) sbt.ResolveException: unresolved dependency: hidden-args#hidden-args_2.11;0.0.1-SNAPSHOT: not found
[error] unresolved dependency: me.rexim#morganey-std_2.11;0.0.1: not found
[error] Total time: 3 s, completed Oct 21, 2016 8:39:08 AM

```
### Expected

The project build successfully.

---

Probably [hidden-args](https://github.com/keddelzz/hidden-args) needs to be published somewhere.

**Comment by rexim** [comment]
See https://github.com/keddelzz/hidden-args/issues/25

**Comment by rexim** [comment]
`morganey % 0.0.1` that is published to JCenter depends on `hidden-args % 0.0.1-SNAPSHOT`. But there is only `hidden-args % 0.0.1` available in the public repos. We need to backport 93c35c2e517180225720a6eff926c9b6af8b5c87 to `0.0.1` to resolve #270 and make it publishable and usable as a thirdparty dependency. And according to http://semver.org/#spec-item-6 we will have to increment the version and publish `morganey % 0.0.2` instead. Unfortunately `morganey % 0.0.1` will be available only via [GitHub Releases](https://github.com/morganey-lang/Morganey/releases/tag/0.0.1). The first Morganey version available via JCenter will be `0.0.2`.

**Comment by rexim** [comment]
On the other hand, does it really matter that much to have `0.0.2`? I mean we could easily make `0.1.0` the first available via JCenter release instead of `0.0.2`. And that doesn't require any backporting and messing with branches at all.

I'm declining `0.0.2` and working towards releasing `0.1.0`.

Note for myself: do not make any serious decisions at night.
